workspaces/cicd-statistics/app-config.yaml

```diff
@@ -0,0 +1,61 @@
+app:
+  title: CI/CD Statistics Example App
+  baseUrl: http://localhost:3000
+
+organization:
+  name: CI/CD Statistics Example
+
+backend:
+  baseUrl: http://localhost:7007
+  listen:
+    port: 7007
+  csp:
+    connect-src: ["'self'", 'http:', 'https:']
+  cors:
+    origin: http://localhost:3000
+    methods: [GET, HEAD, PATCH, POST, PUT, DELETE]
+    credentials: true
+  database:
+    client: better-sqlite3
+    connection: ':memory:'
+
+techdocs:
+  builder: 'local'
+  generator:
+    runIn: 'local'
+  publisher:
+    type: 'local'
+
+auth:
+  providers:
+    gitlab:
```

Comment:
Do we really need GitLab auth? Ideally we would just use Guest auth so that it's easy to work with for development. There is an active PR to add BuildKite to this plugin so using Guest would help there too.